### DISCUSSION TITLE
Property 'hitsPerPage' does not exist on type

### DISCUSSION THREAD

**Discussion by beamercola** [algolia/instantsearch#5657] [Q&A] [answered]
I thought I was following the instructions correctly I swear this used to work

```tsx
export function InstantSearch(props: {
  indexName: string
  children: React.ReactNode
  perPage?: number
  filters?: string
}) {
  return (
    <AlgoliaInstantSearch  searchClient={algolia} indexName={props.indexName}>
      <Configure hitsPerPage={props.perPage ?? 20} filters={props.filters} />
      {props.children}
    </AlgoliaInstantSearch>
  )
}
```

Error:
`Type '{ hitsPerPage: number; filters: string | undefined; }' is not assignable to type 'IntrinsicAttributes & PlainSearchParameters'.`

**Comment by Haroenv** [comment, marked as answer]
This is indeed a new issue caused by the latest version of typescript, you can follow resolution in #5658
